liquibase_mssql.js

```diff
@@ -0,0 +1,91 @@
+const liquibase = require('node-liquibase');
```

Comment:
it's convention to store these sorts of files in an `examples` directory for node:

```
.
|_examples
|_src
|_dist
```

Reply:
Great suggestion.   Done.